src/ChoiceFieldset/ChoiceFieldsetListItem.tsx

```diff
@@ -0,0 +1,75 @@
+import React, {useContext} from 'react'
+import {ComponentProps} from '../utils/types'
+import {uniqueId} from '../utils/uniqueId'
+import ToggleInputLeadingVisual from '../_InputField/ToggleInputLeadingVisual'
+import ChoiceFieldCaption from './ChoiceFieldCaption'
+import ChoiceFieldLabel from './ChoiceFieldLabel'
+import ChoiceFieldsetListContext from './ChoiceFieldsetListContext'
+
+export interface ChoiceFieldProps {
+  /**
+   * Whether the field is ready for user input
+   */
+  disabled?: boolean
+  /**
+   * The unique identifier for this field. Used to associate the label, validation text, and caption text.
+   * If an ID is not provided, one will be automatically generated.
+   */
+  id?: string
+  /**
+   * The value that is being selected
+   */
+  value: string
+}
+
+const ChoiceFieldsetListItem: React.FC<ChoiceFieldProps> = ({children, id, disabled: disabledProp, value}) => {
+  const choiceFieldsetListContext = useContext(ChoiceFieldsetListContext)
+  if (choiceFieldsetListContext === null) {
+    throw new Error('ChoiceFieldsetListContext returned null')
```

Comment:
curious about this, do our users expect us to be throwing native stack errors like this? It'll trigger hits on their observability tooling, that they may not be expecting (or will they?). We use console.warn in other places too, should this be console.error instead?

Reply:
I went with an `Error` because if the context returns null, the component function correctly. I think this will feel natural to consumers.